.gitlab-ci.yml

```diff
@@ -93,13 +93,20 @@ build apps image:
               --build-arg CONAN_REMOTE_URL \
               --build-arg CONAN_USERNAME \
               --build-arg CONAN_PASSWORD \
-              -f Dockerfile.apps
+              -f Dockerfile.apps \
               -t $REPOSITORY_URL:$TAG .
         - docker run --rm -it $REPOSITORY_URL:$TAG
```

Comment:
Der job schlägt [hier fehl](https://gl.ambrosys.de/os-matcher/os-matcher-on-github/-/jobs/202543):
```bash
$ docker run --rm -it $REPOSITORY_URL:$TAG
the input device is not a TTY
Cleaning up file based variables 00:00
ERROR: Job failed: exit code 1
```

Ich vermute das `--it` muss weg oder?
Warum soltest du das interactive starten?